pkg/api/types.go

```diff
@@ -2033,9 +2035,39 @@ const (
 	// Number of Pods that may be running on this Node: see ResourcePods
 )
 
+// ResourceNameList is a collection of ResourceName values.
+type ResourceNameList []ResourceName
```

Comment:
To elaborate, we tend to keep types.go as just holding the structs and all the utility functions should go elsewhere.  It makes it easier to work with the versioned types files if we don't tend to have the utility functions mixed in.